.travis.yml

```diff
@@ -11,14 +11,31 @@ language: julia
 os:
   - linux
   - osx
+  - windows
+
+arch:
+  - x64
+  - x86
 
 julia:
   - "1.0"
   - "1.1"
   - "1.2"
   - "1.3"
+  - "1.4"
+  - "1" # "1" automatically expands to the latest stable release of the form `1.y.z`
```

Comment:
Why quote the numbers?

Reply:
There is a bug in YAML where `1.0` turns in to `1`. Which is not the same thing.

So we need to quote `"1.0"`. And for consistency I quote all numbers as a matter of habit. Otherwise it looks quite weird.

Reply:
I think Travis has a beta feature that fixes this issue. But I'd rather not rely on that. Easiest to quote version numbers, that way we ensure it means what we want it to mean.

Reply:
Are you sure about that? I haven't quoted my `1.0` in TimeZones.jl and https://github.com/JuliaTime/TimeZones.jl/blob/master/.travis.yml#L32 and the CI uses 1.0.1: https://travis-ci.org/github/JuliaTime/TimeZones.jl/jobs/677696077

Reply:
It absolutely used to be the case. See e.g. https://github.com/travis-ci/docs-travis-ci-com/pull/1537#issuecomment-342764863

Late last year, Travis started a new beta "Build Config Validation" that allegedly does not have this issue. I'm assuming that's what is happening on TimeZones.jl. I don't really want to have a `.travis.yml` file that relies on a relatively recent feature in Travis. I'd rather just have a `.travis.yml` file that we know will work correctly regardless.

Reply:
Interesting. It's possible I missed when it was broken.

Reply:
Do we want to include `"1"` as it's currently redundant and will make the CI take longer?

Reply:
Good question. The advantage of "1" is that if we forget to add "1.5" when 1.5 is released, it will still get tested.

What do you think about taking "1.4" off the list? 1.4 will get tested automatically by the "1" entry.

Then when 1.5 is released, we can add "1.4" to the list.

That way, the consequence of forgetting to update the Travis file is that we don't test on the second most recent version. In my mind, that consequence is not as bad as forgetting to test on the most recent version.

Reply:
I can get behind that. The only disadvantage I see is that `"1"` sometimes takes a while to get update and can fall behind the latest release. Having it update eventually though is better here.